server/src/metrics/client_metrics.rs

```diff
@@ -629,13 +630,14 @@ mod test {
     #[test_case(1, 10000, 25; "1 app 10k toggles, will be split into 25 batches")]
     #[test_case(1000, 1000, 7; "1000 apps 1000 toggles, will be split into 7 batches")]
     #[test_case(500, 5000, 15; "500 apps 5000 toggles, will be split into 15 batches")]
-    #[test_case(5000, 1, 17; "5000 apps 1 metric will be split")]
+    #[test_case(5000, 1, 19; "5000 apps 1 metric will be split")]
```

Comment:
I don't know why but the change changed the number of batches. It looks like we did similar updates in the past.

Reply:
Probably because the size increased. the connection_id field name alone is 13 bytes extra per entry.

Reply:
13 bytes for key,
connection-${ulid} is 47 bytes more, so 60 bytes extra per application.